packages/benchmark/bin/n8n-benchmark

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env node
+
+// Check if version should be displayed
+const versionFlags = ['-v', '-V', '--version'];
+if (versionFlags.includes(process.argv.slice(-1)[0])) {
+	console.log(require('../package').version);
+	process.exit(0);
+}
+
+(async () => {
+	const oclif = require('@oclif/core');
+	await oclif.execute({ dir: __dirname });
```

Comment:
Would we pick oclif in cli if starting today? oclif is nice for familiarity, but I wonder if there are better options out there.

Reply:
For a simple tool like this it suffices well enough. Chose it to be consistent with what we are using with n8n.